commands/command_clone.go

```diff
@@ -19,11 +20,28 @@ var (
 	cloneFlags git.CloneFlags
 
 	cloneSkipRepoInstall bool
+
+	// cloneIsDeprecated marks whether or not the clone command is
+	// deprecated. It is false until Git v2.15.0 is released, including the
+	// 'delay' capability.
+	cloneIsDeprecated = false
 )
 
 func cloneCommand(cmd *cobra.Command, args []string) {
 	requireGitVersion()
 
+	if cloneIsDeprecated {
+		msg := []string{
+			"WARNING: 'git-lfs(1) clone' is deprecated and will not be updated",
+			"          with new flags from 'git clone'",
+			"",
+			"'git-clone(1)' has been updated in upstream Git to have comparable",
+			"speeds to 'git-lfs(1) clone'.",
```

Comment:
I would replace `git-lfs(1)` with just `git-lfs` as I imagine `(1)` would confuse most users (just a gut feeling, though).

Reply:
@larsxschneider good call, I removed the `(1)` suffix in 44c230c.